webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -11,66 +11,75 @@
 
 -->
 <web-app xmlns="http://java.sun.com/xml/ns/javaee"
-		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-Instance"
-		 xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
-		 version="3.0">
-	
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-Instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
```

Comment:
Can you redo this file such that it has only the needed changes?

Reply:
@dkayiwa, sorry about that earlier mess.  Intelij sometimes pays deadly games.  But its solved now :slightly_smiling_face: